### PR TITLE
managers: we accidentally disabled addressMgr, bring it back

### PR DESCRIPTION
**Pull request by dorileo**

### CODE REVIEW

File: google_guest_agent/main.go [hunk 1]
```diff
@@ -89,12 +89,14 @@ func closeFile(c io.Closer) {
 func availableManagers() []manager {
 	if runtime.GOOS == "windows" {
 		return []manager{
+			&addressMgr{},
```
Comment (ChaitanyaKulkarni28) [inline google_guest_agent/main.go:92]: since we have a common manager, do you want to keep a list of common managers and append to it based on os type?

Comment (dorileo) [reply google_guest_agent/main.go:92]: Could be, I can change it.

Comment (dorileo) [reply google_guest_agent/main.go:96]: done. thanks.

File: google_guest_agent/main.go [hunk 2]
```diff
@@ -87,18 +87,25 @@ func closeFile(c io.Closer) {
...
 			newWsfcManager(),
 			&winAccountsMgr{},
 			&diagnosticsMgr{},
-		}
+		)
 	}
-	return []manager{
+
+	return append(managers,
+		&addressMgr{},
```
Comment (ChaitanyaKulkarni28) [inline google_guest_agent/main.go:104]: looks like we're appending `addressMgr` here as well. Same in case of windows

Comment (dorileo) [reply google_guest_agent/main.go]: indeed. fixed. thanks.

**Pull request merged**